src/DataCollector/LaravelCollector.php

```diff
@@ -49,7 +49,7 @@ public function getWidgets()
     {
         return [
             "version" => [
-                "icon" => "github",
+                "icon" => "laravel phpdebugbar-fab",
```

Comment:
why the phpdebugbar-fab?

Reply:
After FA 4.x, icons were divided into several sets, example, `fab`(_`phpdebugbar-fab` here_), what does font awesome brands mean

![image](https://github.com/user-attachments/assets/f13a6030-799f-4797-a6ae-7fc9d02eedc3)